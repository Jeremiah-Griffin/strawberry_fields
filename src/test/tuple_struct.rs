use strawberry_fields::strawberry_fields;

#[strawberry_fields(u32)]
struct Dummy(u32, u32);

fn main() {}
