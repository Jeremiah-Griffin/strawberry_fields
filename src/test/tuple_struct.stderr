error: custom attribute panicked
 --> src/test/tuple_struct.rs:3:1
  |
3 | #[strawberry_fields(u32)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: internal error: entered unreachable code: Macro should have checked that it isn't expanding for a tuple stuct, but did not.
